### ISSUE TITLE
Toolkit Name

### ISSUE THREAD

**Issue by MaezrFulldive** [pop-os/libcosmic#577]
[Suggestion] This toolkit is truly the closest thing Linux has to Material You, we need better branding so that when someone wants to make a 'libcosmic' styled application, they aren't running over the namespace. Something like: The Celestial theme would work so that the official cosmic apps do not get overwritten by every community app that pops up on GitHub, and community apps can standardize around cool sounding movement. You could also make a seperate repo for the design spec. If google called Material You the 'android-style', there wouldn't be nearly as many beautiful apps in the lively ecosystem we see today, and to that end.

I implore you to not let this opportunity pass you by like gnome did with Adwaita, DO NOT MISS: Linux's Material You moment.

**Comment by mmstick** [comment]
See the trademark policy: https://github.com/pop-os/cosmic-epoch/blob/master/TRADEMARK.md

Specifically this section on namespacing:

> Third-party applications, applets, and software that integrate with or extend the COSMIC desktop environment must not use the COSMIC trademark in a way that implies official status or endorsement without prior approval from System76. Third-party developers are encouraged to use the "cosmic-ext-" package namespace. These applications and applets may be described as "for the COSMIC™ desktop".

**Comment by MaezrFulldive** [comment]
Right, but if we take my comparison seriously between lib cosmic, and Material YOU, if devs used 'for the android operating system', Material YOU is obviously superior.

All I ask is that you consider making a more brand-able naming convention for the awesome next generation movement that this framework can be.

**Comment by MaezrFulldive** [comment]
Also, I sincerely thank you for listening.
